src/test/data/JsonExerciseTrackerStorageTest/invalidAndValidExerciseExerciseTracker.json

```diff
@@ -3,11 +3,13 @@
     "name": "Valid Exercise",
     "weight": "9482424",
     "sets": "hans@example.com",
-    "reps": "1"
+    "reps": "1",
+    "tagged": "01/01/2022"
```

Comment:
Should be date instead of tagged? If so, should be applied to the other .json files as well!

Reply:
The necessary changes have been made.